### PR TITLE
Task/150753 content change on suggest org card

### PR DESCRIPTION
**Pull request by pablocodeglia**
image

### CODE REVIEW

File: src/modules/shared/pages/innovation/data-sharing-and-support/components/organisation-suggestion-card.component.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import { Component, Input, OnChanges } from '@angular/core';
 
 import { AccessorSuggestionModel, AssessmentSuggestionModel, OrganisationSuggestionModel } from '@modules/stores/innovation/innovation.models';
+import { timingSafeEqual } from 'crypto';
+import { each, filter } from 'lodash';
+import { elementAt } from 'rxjs';
```
Comment (diogo-costa-bjss) [inline src/modules/shared/pages/innovation/data-sharing-and-support/components/organisation-suggestion-card.component.ts:6]: @pablocodeglia I think these imports are not being used.

File: src/modules/shared/pages/innovation/data-sharing-and-support/components/organisation-suggestion-card.component.ts [hunk 2]
```diff
@@ -18,68 +21,58 @@ export class OrganisationSuggestionsCardComponent implements OnChanges {
...
+    });
 
-    // removes duplicate entries
-    const organisations = [...new Set(suggestedOrganisations)];
-    const accessors = [... new Set(accessorsUnits)];
-    return {
-      suggestors: accessors.join(', '),
-      organisations,
-    };
+    filteredSuggestions.forEach(entry => entry.suggestedOrganisationUnits.sort((a,b) => a.name.localeCompare(b.name)));
```
Comment (diogo-costa-bjss) [inline src/modules/shared/pages/innovation/data-sharing-and-support/components/organisation-suggestion-card.component.ts:70]: @pablocodeglia if you prefer, you can add this `sort` directly in line 66 after the `filter`, there is no need to rerun the array.

`this.array.filter(...).sort(...)`

File: src/modules/shared/pages/innovation/data-sharing-and-support/components/organisation-suggestion-card.component.ts (hunk 1 above)
Comment (pablocodeglia) [reply src/modules/shared/pages/innovation/data-sharing-and-support/components/organisation-suggestion-card.component.ts]: You are correct, removed them.

File: src/modules/shared/pages/innovation/data-sharing-and-support/components/organisation-suggestion-card.component.ts (hunk 2 above)
Comment (pablocodeglia) [reply src/modules/shared/pages/innovation/data-sharing-and-support/components/organisation-suggestion-card.component.ts]: Thanks, good call.

**Review by diogo-costa-bjss — APPROVED** [review]

**Pull request merged**